lambda-otel4s/shared/src/main/scala/feral/lambda/otel4s/Attributes.scala

```diff
@@ -75,7 +75,7 @@ object LambdaContextAttributes {
       CloudProvider(CloudProviderValue.Aws.value),
       CloudResourceId(context.invokedFunctionArn),
       FaasInstance(context.logStreamName),
-      FaasMaxMemory(context.memoryLimitInMB.toLong),
+      FaasMaxMemory(context.memoryLimitInMB.toLong * 1024 * 1024),
```

Comment:
According to the [spec](https://opentelemetry.io/docs/specs/semconv/attributes-registry/faas/) we must use bytes there.